patches/net/minecraft/client/particle/Particle.java.patch

```diff
@@ -1,21 +1,29 @@
 --- a/net/minecraft/client/particle/Particle.java
 +++ b/net/minecraft/client/particle/Particle.java
-@@ -245,6 +_,18 @@
-         return Optional.empty();
-     }
+@@ -19,6 +_,7 @@
+ @OnlyIn(Dist.CLIENT)
+ public abstract class Particle {
+     private static final AABB INITIAL_AABB = new AABB(0.0, 0.0, 0.0, 0.0, 0.0, 0.0);
++    public static final AABB INFINITE_EXTENT_AABB = net.neoforged.neoforge.client.extensions.IBlockEntityRendererExtension.INFINITE_EXTENT_AABB;
```

Comment:
I don't really like this cross bleed of api usage. I would suggest moving the field

Reply:
What is a good shared location to put it? I don't think we currently have a constants class for rendering stuff.